README.md

```diff
@@ -1,18 +1,75 @@
 # SwissTournament
-A Swiss Tournament Pairing Generator written for the [2nd Muenchener Kaffeehaus Chess Tournament](https://www.meetup.com/Chess-Munich/events/rqwkqsyccpbgb/).
+A Swiss Tournament Pairing Generator written for the [3rd Muenchener Kaffeehaus Chess Tournament](https://www.meetup.com/Chess-Munich/events/svkrqsyccqbdb/).
 
-# How to use
-- To the `input.txt` file, add the player names after a `P` and a newline, one per line.
-- Then add the round results after a `R` and a newline, in the format `<White Player name>:<Black Player name>:<W/B/D>` one per line.
-- If the Tournament does not have any recorded results, you can already generate the pairings for the first round by running
-  
-  ```python
-  python generate.py
-  ```
-  Open `tournament.md` in any markdown viewer to view the standings, pairings for the next round and the results of the previous rounds.
-- Input the results of the current round by changing the `NP`s to `W` for a white win, `B` for a black win and `D` for a draw. 
-- Run `python generate.py` again to generate the current standings and pairings for the next round. Pairing generation is stopped when the maximum number of rounds is reached.
+## Pre-requirements
+- Install dependencies to your environment 
+```
+pip install -r requirements.txt
+```
+## How to use
+- Execute the cli to display the help with the usage
+```
+python swiss_tournament.py
+```
+## Test that everything is working as expected
+- Run all tests from the project
+```
+pytest
+```
+NOTE: Tests will try to generate files under directory `/tmp/`
+## Help for each command
+- How to generate a tournament
+```
+python swiss_tournament.py create-tournament --help
```

Comment:
I would rather print the contents of the `--help` command

Reply:
good point too